docs/pages/service/getStarted.mdx

```diff
@@ -0,0 +1,115 @@
+---
+title: 'Get started'
+---
+
+import { Tab, Tabs } from 'nextra-theme-docs'
+
+# Get started
+
+## Step 1: Identify the Desired Service
+
+Start by determining which service you want to respond to. A list of available services can be found in the relevant sections of the documentation.
+
+## Step 2: Establish a Service Endpoint
+
+Create a publicly accessible HTTP endpoint to receive incoming requests using the POST method. This endpoint should be able to return specific payloads based on the requested service.
+
+<Tabs items={['Next.js', 'Express']}>
+<Tab>
+```ts filename="./pages/api/service.ts"
```

Comment:
It's not clear that we recommend to implement one API per type of service.
```suggestion
```ts filename="./pages/api/SERVICE_TYPE.ts"
```